src/main/java/www/purple/mixxy/controllers/ComicController.java

```diff
@@ -17,48 +23,85 @@
 public class ComicController {
 
 	@Inject
-    private ComicDao comicDao;
-	
-  public Result remix() {
-    return Results.TODO();
-  }
-
-  public Result edit() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result newWork() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result likes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result remixes() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result root() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result parent() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result delete() {
-    return Results.TODO();
-  }
-
-  @FilterWith(JsonEndpoint.class)
-  public Result update() {
-    return Results.TODO();
-  }
+	private ComicDao comicDao;
+
+	@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+	public Result remix() {
+		return Results.TODO();
+	}
+
+	public Result edit() {
+		return Results.TODO();
+	}
+
+	public Result comicShow(@PathParam("id") Long id) {
```

Comment:
The URL schema is `/{user}/{comic-slug}`.  I suppose we can support this if it's easy, but nicer urls == nicer website.